terraform/modules/matrix-service/redshift/s3.tf

```diff
@@ -1,8 +1,8 @@
-resource "aws_s3_bucket" "matrix_service_redshift_preload" {
-  bucket = "dcp-matrix-service-redshift-preload-${var.deployment_stage}"
+resource "aws_s3_bucket" "matrix_service_preload" {
+  bucket = "dcp-matrix-service-preload-${var.deployment_stage}"
 
   lifecycle_rule {
-    id      = "matrix_service_redshift_preload_expiration"
+    id      = "matrix_service_preload_expiration"
 
     expiration {
```

Comment:
Why do we have an expiration of 7 days?

Reply:
Do you suggest it should be lower or higher or have none? The staged PSVs from a notification has little use beyond debugging purposes. A better argument can be made for staged PSVs from a full load. Maybe a full load should use a different bucket than notifications where the former has a longer expiration than the latter (to preserve a sort of a snapshot to perform expedited reloads)?

Reply:
Alternatively, it could be a single bucket with a longer (30 day?) expiration

Reply:
Hmm, yea, I prefer 30 days if we are keeping something around for debugging purposes. Treat it like logs in a way